assets/javascripts/initializers/group-tag-associations.js.es6

```diff
@@ -0,0 +1,18 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "group-tag-associations",
+  initialize() {
+    withPluginApi("0.11.0", (api) => {
+      api.modifyClass("model:group", {
+        asJSON() {
+          const attrs = this._super(...arguments);
+          if (this.associated_tags) {
+            attrs["associated_tags"] = this.associated_tags;
```

Comment:
```suggestion
            attrs.associated_tags = this.associated_tags;
```
I thought we had a linting rule for this, but we needn't use quotes when the object property has no spaces or weird characters.